app/src/main/java/com/globant/harrypotterapp/adapter/CharactersAdapter.kt

```diff
@@ -8,7 +8,11 @@ import com.globant.harrypotterapp.R
 import com.globant.domain.entity.Character
 import com.globant.harrypotterapp.databinding.FragmentCharactersCardViewBinding
 
-class CharactersAdapter : RecyclerView.Adapter<CharactersAdapter.ViewHolder>() {
+class CharactersAdapter(private val onCharacterClick: OnCharacterClick) : RecyclerView.Adapter<CharactersAdapter.ViewHolder>() {
+
+    interface OnCharacterClick {
```

Comment:
this interface should be outside the class

Reply:
You are right! Refactored!